community/cypher/frontend-3.4/src/test/scala/org/neo4j/cypher/internal/frontend/v3_4/ast/MultipleGraphClauseSemanticCheckingTest.scala

```diff
@@ -35,16 +35,44 @@ class MultipleGraphClauseSemanticCheckingTest
 
   implicit val parser: Rule1[Query] = Query
 
-  test("allows bound nodes in single node pattern GRAPH OF") {
+  test("finds semantic errors in clauses inside CONSTRUCT") {
     parsing(
-      """MATCH (a:Swedish)
-        |RETURN GRAPH result OF (a)""".stripMargin) shouldVerify { result: SemanticCheckResult =>
+      """CONSTRUCT { REMOVE a.prop }
+        |RETURN 1 as a
+      """.stripMargin) shouldVerify { result: SemanticCheckResult =>
+
+      result.errorMessages should equal(Set("Variable `a` not defined"))
+    }
+  }
 
+//  test("does not allow @ in CREATE") {
+//    parsing("MATCH (a) CREATE (@a)") shouldVerify { result: SemanticCheckResult =>
+//
+//      result.errorMessages should equal(Set("@ equivalence is only allowed in MERGE"))
+//    }
+//  }
+
+//
+//  test("sets the working with CONSTRUCT GRAPH") {
+//    parsing(
+//      """CONSTRUCT GRAPH foo.bar.baz {
+//        |  CREATE ()
+//        |}
+//        |RETURN 1 as a
+//      """.stripMargin) shouldVerify { result: SemanticCheckResult =>
+//      result.state.workingGraph should equal(Some(ast.QualifiedGraphName(List("foo", "bar", "baz"))))
+//      result.errors shouldBe empty
+//    }
+//  }
+
+  test("can conclude with RETURN GRAPH") {
+    parsing(
+      """RETURN GRAPH""".stripMargin) shouldVerify { result: SemanticCheckResult =>
       result.errors shouldBe empty
     }
   }
 
-  test("allows bound relationships in GRAPH OF") {
+  ignore("allows bound relationships in GRAPH OF") {
```

Comment:
These tests are ignored. Are they meant to be deleted or addresses in a different PR?